wire-ios/Tests/TestPlans/UnitTests.xctestplan

```diff
@@ -0,0 +1,25 @@
+{
+  "configurations" : [
+    {
+      "id" : "F8620569-DB08-4783-A3B9-17B0FBEBF254",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "language" : "en",
+    "region" : "US"
+  },
+  "testTargets" : [
```

Comment:
is it the equivalent of the GermanLocaleStrings plan. In this plan we have a couple of tests that need to be run on German locale, probably GermanLocaleStrings needs the testTarget Wire-iOS UnitTests"

Reply:
I have copied this from the "AllTests.xctestplan", so I assume that's correct. There is another `GermanLocaleTest.xctestplan`, that is configured for it.
<img width="1413" alt="Screenshot 2023-11-14 at 19 01 42" src="https://github.com/wireapp/wire-ios/assets/7196958/bbfac918-6ea4-477a-955f-27d5cf3961b9">